packages/cloudflare/src/index.ts

```diff
@@ -26,6 +26,11 @@ import { wasmModuleLoader } from './utils/wasm-module-loader.js';
 export type { AdvancedRuntime } from './entrypoints/server.advanced.js';
 export type { DirectoryRuntime } from './entrypoints/server.directory.js';
 
+interface CfRuntimeOptions {
+	mode: 'off' | 'local' | 'remote';
+	persistTo: string;
+}
```

Comment:
A suggestion. If `persistTo` is relevant only when using `local`, maybe we make types better:

```ts
type CfRuntimeOptions = 
	{ mode: 'off' } |
	{ mode: 'remote' } |
	{ mode: 'local', persistTo: string }

```

Reply:
Yeah don't overthink, it's just a suggestion :) as long as the code works, that's what matters